orangewidget/widget.py

```diff
@@ -330,12 +334,40 @@ def sizeHint(self):
         def minimumSizeHint(self):
             return self._adjusted_size(attrgetter("minimumSizeHint"))
 
+        def setSizes(self, sizes):
+            super().setSizes(sizes)
+            if len(sizes) == 2:
+                self.handle(1).setControlAreaShown(bool(sizes[0]))
+
         def createHandle(self):
             """Create splitter handle"""
             return self._Handle(
                 self.orientation(), self, cursor=Qt.PointingHandCursor)
 
         class _Handle(QSplitterHandle):
+            def __init__(self, *args, **kwargs):
+                super().__init__(*args, **kwargs)
+                self.controlAreaShown = True
+
+            def setControlAreaShown(self, shown):
+                self.controlAreaShown = shown
+                self.update()
+
+            def paintEvent(self, event):
+                super(QSplitterHandle, self).paintEvent(event)
+                painter = QPainter(self)
+                painter.setPen(QColor(160, 160, 160))
+                w = self.width() - 2
+                if self.controlAreaShown:
+                    x0, x1 = 1, w - 1
+                else:
+                    x0, x1 = w - 1, 1
```

Comment:
Starting 2 pixels away from the left border looks better for me than 1.
+ as @janezd noticed, the above code does not center (1-3 margins, instead of 2-2)